test/EmbedIO.Tests/BasicAuthenticationModuleTest.cs

```diff
@@ -30,6 +30,7 @@ public async Task RequestWithValidCredentials_ReturnsOK()
         {
             var response = await MakeRequest(UserName, Password).ConfigureAwait(false);
             Assert.AreEqual(HttpStatusCode.OK, response.StatusCode, "Status Code OK");
+            Assert.AreEqual("Basic realm=\"/\" charset=UTF-8", response.Headers.WwwAuthenticate.ToString());
```

Comment:
It's a good thing you added this check, but it should be in its own method, maybe named `RequestWithValidCredentials_ReturnsValidWWWAuthenticateHeader`.